pimd/pim_bsm.c

```diff
@@ -1371,7 +1371,7 @@ int pim_bsm_process(struct interface *ifp, struct ip *ip_hdr, uint8_t *buf,
 	}
 
 #if PIM_IPV == 4
-	if (ip_hdr->ip_dst.s_addr == qpim_all_pim_routers_addr.s_addr)
+	if (!pim_addr_cmp(sg->grp, qpim_all_pim_routers_addr))
 #else
 	if (0)
```

Comment:
We should have the same for IPV6 as well.
Why we are having if(0) for PIMV6 ?

Reply:
The code was previously disabled and isn't changed by this PR.

(We don't have BSM enabled for IPv6 yet so it doesn't really matter either way, the file isn't even compiled…)